bfdd/bfd_packet.c

```diff
@@ -215,51 +215,70 @@ static int ptm_bfd_process_echo_pkt(struct bfd_vrf_global *bvrf, int s)
 void ptm_bfd_snd(struct bfd_session *bfd, int fbit)
 {
 	struct bfd_pkt cp;
+	struct bfd_pkt *pst_bfd_pkt = NULL;
```

Comment:
added goto

I have found this debug useful am sure it will be useful while debugging any other issue as well.
Keeping the debug message since it is can be turned on/off using "bfd debug" cli